src/client/app/components/visual-unit/CreateVisualUnitModalMapComponent.tsx

```diff
@@ -0,0 +1,219 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file and CreateCikVisualMapComponent are very similar. Would it be possible to combine them and use passed props to get the one you want? Would make upkeep easier if you agree.